projects/noloan/admin/src/main/java/hasadna/noloan/admin/app/SuggetionRecyclerAdapter.java

```diff
@@ -2,42 +2,100 @@
 
 import android.os.Handler;
 import android.os.Looper;
+
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.TextView;
 import android.widget.Toast;
 
-import hasadna.noloan.admin.app.firestore.FirestoreClient;
+import java.util.ArrayList;
+
+import hasadna.noloan.firebase.DbMessages;
+import hasadna.noloan.firebase.FirestoreClient;
 import hasadna.noloan.protobuf.SmsProto.SmsMessage;
 
 public class SuggetionRecyclerAdapter
     extends RecyclerView.Adapter<SuggetionRecyclerAdapter.RecyclerViewHolder> {
 
-  DbMessagesHolder DbMessages;
+  DbMessages dbMessages;
```

Comment:
This is a question I asked myself too...

If DbMessages is a Singleton and the same instance is been used thru out all of the app...

Should it be declared as a Class member for classes that uses it?
Or just be "fetched" with `DbMessages.getInstance()` when ever needed ?

I don't think it actually matters much using it either way, but this thought still raises up...

Reply:
You are right, it shouldn't. fixed